src/buildercore/trop.py

```diff
@@ -205,18 +200,14 @@ def ec2instance(context, node):
     buildvars_serialization = bvars.encode_bvars(buildvars)
 
     odd = node % 2 == 1
-    if odd:
-        subnet_id = lu('project.aws.subnet-id')
-    else:
-        subnet_id = lu('project.aws.redundant-subnet-id')
-
+    subnet_id = lu('aws.subnet-id') if odd else lu('aws.redundant-subnet-id')
     clean_server = _read_script('.clean-server.sh.fragment')
     project_ec2 = {
-        "ImageId": lu('project.aws.ec2.ami'),
-        "InstanceType": context['ec2']['type'], # t2.small, m1.medium, etc
+        "ImageId": lu('ec2.ami'),
+        "InstanceType": lu('ec2.type'), # "t2.small", "m1.medium", etc
         "KeyName": Ref(KEYPAIR),
         "SecurityGroupIds": [Ref(SECURITY_GROUP_TITLE)],
-        "SubnetId": subnet_id, # ll: "subnet-1d4eb46a"
+        "SubnetId": subnet_id, # "subnet-1d4eb46a"
         "Tags": instance_tags(context, node),
 
```

Comment:
the `trop` module in general seems to have been simplified

Reply:
a little nip and tuck